Translations/pt_BR.po

```diff
@@ -1419,7 +1410,7 @@ msgstr "A salvação vem à custa de sabedoria"
 #. TRANSLATORS: Shrine Text. Keep atmospheric. :)
 #: Source/error.cpp:65
 msgid "Mysteries are revealed in the light of reason"
-msgstr "Mistérios são revelados à luz da razão"
+msgstr "Os mistérios são revelados na luz da razão"
```

Comment:
Also disagree. Original is better here.

Reply:
?
```suggestion
msgstr "Mistérios são revelados à luz da razão"
```

Reply:
@AJenbo this translation should be reverted, the original was better
```
Os mistérios são revelados na luz da razão
```

Reply:
That didn't really clear up for me, your comment has the one from the update in it afak, but it's not as a suggestion, and I don't feel like you confirmed mine, but I think it's what matches what you are saying, but then why did you say it and not just "yes"....

Reply:
@AJenbo oops, my bad I copy+paste the wrong sentence. I updated my comment.

Reply:
You also didn't mark it as a suggestion :)